hello-oboe/app/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
remove this .gitignore file, and incorporate it into the parent's .gitignore file.